skypy/power_spectrum/_halofit.py

```diff
@@ -98,18 +99,45 @@ def halofit(wavenumber, redshift, linear_power_spectrum,
 
     Examples
     --------
+
+    This will return the halofit power spectrum for a given wavenumber,
+    redshift and linear power spectrum:
+
     >>> import numpy as np
     >>> from astropy.cosmology import FlatLambdaCDM
-    >>> kvec = np.array([1.00000000e-04, 1.01000000e+01])
-    >>> zvalue = 0.0
-    >>> pvec = np.array([388.6725682632502, 0.21676249605280398])
     >>> cosmo = FlatLambdaCDM(H0=67.04, Om0=0.21479, Ob0=0.04895)
+
+    The Takahashi halofit power spectrum in :math:`Mpc^3` at redshift 0 and at
+    wavenumers :math:`k=0.0001/Mpc` and :math:`k=10.1/Mpc`:
+
+    >>> zvalue = 0.0
+    >>> kvec = np.array([1.00000000e-04, 1.01000000e+01])
+    >>> pvec = np.array([[388.6725682632502, 0.21676249605280398]])
     >>> halofit(kvec, zvalue, pvec, cosmo, _takahashi_parameters)
```

Comment:
Even though it was like that from before, I think this example would better use `halofit_takahashi` since the `_takahashi_parameters` are not exported from the module.